network/framework/src/protocols/health_checker/mod.rs

```diff
@@ -362,30 +361,13 @@ impl<NetworkClient: NetworkClientInterface<HealthCheckerMsg> + Unpin> HealthChec
                     .get_peer_failures(peer_id)
                     .unwrap_or(0);
                 if failures > self.ping_failures_tolerated {
+                    // TODO deprecate: this no longer disconnects peers, only responds to pings
```

Comment:
It probably makes sense for us to remove the logic that still sends ping messages? i.e., just make this code to respond to messages?

Reply:
This method is for handling ping responses though, not for sending pings? Maybe I'm misunderstanding. @JoshLind

Reply:
Aah, yeah, I was just commenting here because of the TODO 😄